package.json

```diff
@@ -9,7 +9,7 @@
     "font-awesome": "~4.7.0",
     "foundation-sites": "~6.3.0",
     "github-markdown-css": "~2.4.1",
-    "jquery": "^2.0",
+    "jquery": "^3.0.0",
```

Comment:
No reason other than making the minimum amount of changes needed to address the story. I'll see if there are any issues with the higher version.